### PR TITLE
this change throws an exception - after test it will be removed

### PR DESCRIPTION
**Pull request by mbiarnes**
this is only for testing - will be removed later on

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

File: job-dsls/jobs/dailyBuild_pipeline.groovy [hunk 1]
```diff
@@ -11,7 +11,7 @@ def organization=Constants.GITHUB_ORG_UNIT
 def deployDir="deploy-dir"
 def m2Dir = Constants.LOCAL_MVN_REP
 
-String EAP7_DOWNLOAD_URL = "http://download.devel.redhat.com/released/JBoss-middleware/eap7/7.2.0/jboss-eap-7.2.0.zip"
+String EAP7_DOWNLOAD_URL = "http://download.devel.redhat.com/released/JBoss-middleware/eap7/7.2.0/jboss-eap-7.2.0.zi
```
Comment (mareknovotny) [inline job-dsls/jobs/dailyBuild_pipeline.groovy]: there is missing 'p' in zip ;)

File: job-dsls/jobs/dailyBuild_prod_pipeline.groovy [hunk 2]
```diff
@@ -14,7 +14,7 @@ def folderPath="daily-build-prod"
 def dailyProdBuild='''
 pipeline {
     agent {
-        label 'kie-linux&&kie-rhel7&&kie-mem24g'
+        label 'kie-linux&&kie-rhel7&&kie-mem24g
```
Comment (mareknovotny) [inline job-dsls/jobs/dailyBuild_prod_pipeline.groovy]: missing ending quote ?

File: job-dsls/jobs/deploy_jobs.groovy [hunk 3]
```diff
@@ -4,7 +4,7 @@
 import org.kie.jenkins.jobdsl.Constants
 
 def final DEFAULTS = [
-        ghOrgUnit              : Constants.GITHUB_ORG_UNIT,
+        ghOrgUnit              : Constants.GITHUB_ORG_UNIT
```
Comment (mareknovotny) [inline job-dsls/jobs/deploy_jobs.groovy]: missing comma

File: job-dsls/jobs/downstream_pr_jobs.groovy [hunk 4]
```diff
@@ -8,7 +8,7 @@ def final DEFAULTS = [
         ghOrgUnit              : Constants.GITHUB_ORG_UNIT,
         branch                 : Constants.BRANCH,
         timeoutMins            : 600,
-        label                  : "kie-rhel7 && kie-mem24g",
+        label                  : "kie-rhel7 && kie-mem24g"
```
Comment (mareknovotny) [inline job-dsls/jobs/downstream_pr_jobs.groovy]: missing comma

**Comment by mareknovotny** [comment]
:) i get it is just for testing purpose :1st_place_medal:

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Pull request closed without merging**